From: Russ Anderson <rja@sgi.com>
Subject: Add partition id, coherence id, and region size to UV
References: bnc#442455

Add partition id, coherence id, and region size to UV.

The SGI xp drivers (drivers/misc/sgi-xp) are used on both
sn (Itanium) and uv (Tukwilla).  Using the same names
(sn_partition_id, sn_coherency_id, sn_region_size)
simplifies the driver code.


Signed-off-by: Russ Anderson <rja@sgi.com>
Acked-by: Bernhard Walle <bwalle@suse.de>

---

 arch/ia64/uv/kernel/setup.c |    6 ++++++
 1 file changed, 6 insertions(+)

--- a/arch/ia64/uv/kernel/setup.c
+++ b/arch/ia64/uv/kernel/setup.c
@@ -19,6 +19,12 @@ EXPORT_PER_CPU_SYMBOL_GPL(__uv_hub_info)
 
 #ifdef CONFIG_IA64_SGI_UV
 int sn_prom_type;
+long sn_partition_id;
+EXPORT_SYMBOL(sn_partition_id);
+long sn_coherency_id;
+EXPORT_SYMBOL_GPL(sn_coherency_id);
+long sn_region_size;
+EXPORT_SYMBOL(sn_region_size);
 #endif
 
 struct redir_addr {
